.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,78 @@
+name: Bug Report
+description: File a bug report to help improve qBittorrent user experience.
+body:
+  - type: markdown
+    attributes:
+     value: |
+      #### ADVISORY
+       "We do not support any versions older than the current release series" or 3rd party/forked versions e.g. `portableapps`/`Enhanced Edition`etc.
+       It is the responsibility of the user to **redact any sensitive information** in logs/config files.
+  - type: checkboxes
+    attributes:
+      label: Prerequisites
+      description: |
+       To rule out invalid issues, confirm and check each one of the required checkboxes.
+      options:
+      - label: I have read the **issue reporting section** in the [contributing guidelines](https://github.com/qbittorrent/qBittorrent/blob/master/CONTRIBUTING.md), so I know how to submit a good bug report with the required information.
+        required: true
+      - label: I have verified that the **issue is not fixed and is reproducible** in the **[latest version](https://www.qbittorrent.org/download.php).**
+        required: true
+      - label: (optional but recommended) I have verified that the **issue is not fixed and is reproducible** in the **[latest CI build](https://github.com/qbittorrent/qBittorrent/actions/workflows/ci.yaml?query=branch%3Amaster+event%3Apush).**
+        required: false
+      - label: I have **checked the [frequent/common issues list](https://github.com/qbittorrent/qBittorrent/projects/2)** and **searched** the issue tracker for similar bug reports (including closed ones) **to avoid posting a duplicate.**
+        required: true
+      - label: This report is **not a support request or question**, both of which are better suited for either the [discussions section](https://github.com/qbittorrent/qBittorrent/discussions), [forum](https://qbforums.shiki.hu/), or [subreddit](https://www.reddit.com/r/qBittorrent/). The [wiki](https://github.com/qbittorrent/qBittorrent/wiki) did not contain a suitable solution either.
+        required: true
+      - label: I have taken note of the **(ADVISORY)** & if relevant to issue or if asked by a **team member** will add the settings file and log(s) in the **Logs & settings file(s)** section at the bottom of the report. See [this wiki page](https://github.com/qbittorrent/qBittorrent/wiki/Frequently-Asked-Questions#Where_does_qBittorrent_save_its_settings) if you're not sure where to find them.
```

Comment:
The reporter should know which parts are required and which are optional, like the 3rd checkbox (optional). Same goes for the fields below, as a placeholder maybe?